.github/workflows/r_valgrind.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   test-r-valgrind:
     name: r-package (ubuntu-latest, R-devel, valgrind)
-    timeout-minutes: 120
+    timeout-minutes: 300
```

Comment:
Is `300` really needed? Maybe try with `180` for the first time?

Reply:
ok 180 is fine

```suggestion
    timeout-minutes: 180
```